lib/Braintree/OAuthGateway.php

```diff
@@ -61,7 +61,8 @@ private function _verifyGatewayResponse($response)
 
     public function _mapError($result)
     {
-        $error = $result->errors->deepAll()[0];
+        $errors = $result->errors->deepAll();
+        $error = $errors[0];
```

Comment:
Should we ensure that `$errors` is an array and has one value to prevent notices?